Transfer/Sources/ScheduleTransferSectionData.swift

```diff
@@ -67,14 +67,17 @@ struct ScheduleTransferForeignExchangeData: ScheduleTransferSectionData {
                 let destinationCurrency = foreignExchange.destinationCurrency,
                 let rate = foreignExchange.rate {
                 let sourceAmountFormatted = String(format: "%@ %@",
-                                                   sourceAmount.format(with: sourceCurrency),
+                                                   sourceAmount.format(withActualDecimal: sourceCurrency),
                                                    sourceCurrency)
                 let destinationAmountFormatted = String(format: "%@ %@",
-                                                        destinationAmount.format(with: destinationCurrency),
+                                                        destinationAmount
+                                                            .format(withActualDecimal: destinationCurrency),
                                                         destinationCurrency)
-                let rateFormatted = String(format: "%@ = %@",
-                                           "1".format(with: sourceCurrency),
-                                           rate.format(with: destinationCurrency))
+                let rateFormatted = String(format: "%@ %@ = %@ %@",
```

Comment:
API returns formatted response so no need to format any value in this class. SO remove format from here. Just leave it as  `String(format: "%@ %@ = %@ %@",
                                           "1".format(with: sourceCurrency),
                                           sourceCurrency,
                                           rate,
                                           destinationCurrency)`